forge-gui-mobile/src/forge/adventure/util/Config.java

```diff
@@ -58,7 +56,17 @@ static public Config instance()
     private Config() {
 
         String path= resPath();
-         adventures = new File(GuiBase.isAndroid() ? ForgeConstants.ADVENTURE_DIR : path + "/res/adventure").list();
+        FilenameFilter planesFilter = new FilenameFilter() {
+            @Override
+            public boolean accept(File file, String s) {
+                if (!s.contains(".") && !s.equals(commonDirectoryName)) {
```

Comment:
Does file.isdirectory() works?

Reply:
No, I tried that. Here, "file" is actually referring to the adventure directory, and "s" is a string from the list of filenames.